File: BEE 1015.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
// var lines = input.split('\n');
var sep = [' ', '\n'];
// console.log(separators.join('|'));
var line = input.split(new RegExp(sep.join('|'), 'g'));

var x1 = parseFloat(line.shift());
var y1 = parseFloat(line.shift());

var x2 = parseFloat(line.shift());
var y2 = parseFloat(line.shift());

var distance = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2));

console.log(distance.toFixed(4));

File: BEE 1015.py
from math import sqrt, pow

x1, y1 = map(float, input().split())
x2, y2 = map(float, input().split())

distance = sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2))

print("{:.4f}".format(distance))